packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -30,6 +31,18 @@ export const AttestationRequestType = t.type({
 
 export type AttestationRequest = t.TypeOf<typeof AttestationRequestType>
 
+const ATTESTATION_EXPIRY_TIMEOUT_MS = 60 * 60 * 24 * 100 // 1 day
```

Comment:
I think you're off by a factor of 10 here

Reply:
nice catch